### PR TITLE
{bio,devel,tools}[foss/2019b] variant_tools v3.1.3, PycURL v7.43.0.5, PyZMQ v18.1.1 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
bear-pg0211u14b.bear.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/9e4dc798d90398f2a844fb8c51d30213 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
bber0501u03a.bb2.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/38a60573a41aa2c725e85bb738486891 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
bber0501u03a.bb2.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/f71bb43780df5fa777fb7c4ebeb71fc5 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
bear-pg0211u14a.bear.cluster - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/873aeb083129bd3a9fbd453f8ff2b49b for a full test report.

File: easybuild/easyconfigs/p/PyZMQ/PyZMQ-18.1.1-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'PyZMQ'
+version = '18.1.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.zeromq.org/bindings:python'
+description = """Python bindings for ZeroMQ"""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
```
Comment (smoors) [inline easybuild/easyconfigs/p/PyZMQ/PyZMQ-18.1.1-foss-2019b-Python-3.7.4.eb:10]: @branfosj why not move this down to `GCCcore` toolchain?

**Review by smoors — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/PyZMQ/PyZMQ-18.1.1-foss-2019b-Python-3.7.4.eb (hunk 1 above)
Comment (branfosj) [reply easybuild/easyconfigs/p/PyZMQ/PyZMQ-18.1.1-foss-2019b-Python-3.7.4.eb]: I've moved it.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/c4d8cf6aca3d0b66f34b81bec9d6cba4 for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 7 out of 7 (3 easyconfigs in this PR)
node350.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/740ae76a6b5d60031d104eea013a585a for a full test report.

**Comment by smoors** [comment]
@boegelbot please test @ generoso

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @branfosj!

**Pull request merged**